WireAnalytics/Sources/WireAnalytics/AnalyticEvents/RestoreBackupSucceeded.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+public struct RestoreBackupSucceeded: AnalyticEvent {
```

Comment:
Perhaps we could have a single `RestoreBackupAnalyticsEvent` that has a `result` property, so it can handle both success and failure outcomes?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/7b912f53688b980081660d1eb3711b27aa607b8e